broker/beam/value_added/value_added.py

```diff
@@ -197,7 +217,15 @@ def run(PROJECTID, sources, sinks, pipeline_args, salt2_configs):
         help="Google Cloud Platform project name.\n",
     )
     parser.add_argument(
-        "--source_PS_ztf",
+        "--SURVEY",
+        help="Survey this broker instance is associated with.\n",
```

Comment:
I think of "The Broker" has the entire thing.  Is the nomenclature here that "broker instance" is a per-survey thing?

Reply:
> I think of "The Broker" has the entire thing.

I also think of it that way... but there is usually (for me) more than one of them.

> Is the nomenclature here that "broker instance" is a per-survey thing?

Partially, yes. Multiple end-to-end brokers can be running in parallel, each ingesting a different survey and/or running different versions of the code. So I needed a way to differentiate them and settled on calling each one a "broker instance" (even recognizing the potential for confusion because of Google's nomenclature of a Compute Engine "instance", aka a VM). 

I refer to "broker instance"s throughout the docs (and try to clarify what it means in the "Broker Instance Keywords" doc), but this might be confusing to people. I am also not necessarily consistent... at times when I feel like there is no confusion I simply refer to "the broker" when I mean "the broker instance". Maybe people expect that multiple instances can run in parallel (or, more likely, they just don't think about running more than one) and so there is no need to differentiate in the first place? Or perhaps "the broker" really already means "the broker instance" and I've just complicated it unnecessarily?

I'm glad you brought this up, and I think it's worth a longer conversation. I wrestled with it, but have never felt like I've come to a good resolution.